blocks/footer/footer.css

```diff
@@ -1,14 +1,63 @@
 footer {
-  padding: 2rem;
-  background-color: var(--overlay-background-color);
-  font-size: var(--body-font-size-s);
+  --footer-text-color: #fff;
+
+  color: var(--footer-text-color);
+  background-color: #000;
+  font-size: var(--body-font-size-xxs);
+  line-height: 1rem;
+  font-family: MCQGlobal-Regular-Heavy,Helvetica,Arial,sans-serif;
```

Comment:
variable name to be used : var(--body-font-family)
https://github.com/hlxsites/macquarie-help-centre/blob/main/styles/styles.css#L23

Reply:
@asthabh23, the font is actually different than `--body-font-family` (`MCQGlobal-Regular-Heavy` vs `MCQGlobal-Regular-Normal`) and I was skeptical to create a variable for each variation in https://github.com/hlxsites/macquarie-help-centre/blob/main/styles/lazy-styles.css 

Do you recommend to use `--body-font-family`  everywhere?

Reply:
added `--body-font-family-regular-heavy`